index.html

```diff
@@ -1,4 +1,414 @@
-<meta http-equiv="refresh" content="0; url=./latest" />
-<h1>Astarte Documentation</h1>
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="assets/dist/styles.css" />
+    <title>Documentation for Astarte</title>
+  </head>
+  <body>
+    <section class="container-fluid text-center w-75 m-auto">
+      <img
+        class="img-fluid col-8 col-sm-6 col-md-5 col-lg-4 col-xl-2 my-5"
+        src="assets/astarte-k.png"
+        alt="astarte-kubernetes"
+      />
+      <div class="text-center container-lg mx-sm-auto mb-4">
+        <h1 class="text-center text-secondary fs fw-bold mb-4">
+          Astarte Documentation
+        </h1>
+        <p class="text-body-tertiary fs-5">
+          A Data Orchestration Platform focused on IoT. Astarte manages your
+          Devices, organises their data, and automatically orchestrates your
+          Data Science Pipelines.
+        </p>
+      </div>
+      <a href="https://github.com/astarte-platform">
+        <img
+          class="rounded img-fluid col-12 col-sm-12 col-md-12 col-lg-12 col-xl-12"
```

Comment:
if we're using the same number of columns on all breakpoints, you only need to specify the smallest (in this instance `col-12`)